ProjectHeads/App.Head.props

```
@@ -40,7 +40,42 @@
       </ItemGroup>
     </When>
   </Choose>
-
+  <Choose>
+    <When Condition="'$(ToolkitSettingsControlsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Controls.SettingsControls" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('SettingsControls')) == 'false'">
+      <ItemGroup>
+        <ProjectReference Include="$(ToolkitSettingsControlsSourceProject)"/>
+      </ItemGroup>
+    </When>
+  </Choose>
+  <Choose>
+    <When Condition="'$(ToolkitExtensionsSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Extensions" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('Extensions')) == 'false'">
+      <ItemGroup>
+        <ProjectReference Include="$(ToolkitExtensionsSourceProject)"/>
+      </ItemGroup>
+    </When>
+  </Choose>
+   <Choose>
+    <When Condition="'$(ToolkitTriggersSourceProject)' == ''">
+      <ItemGroup>
+        <PackageReference Include="CommunityToolkit.$(DependencyVariant).Triggers" Version="8.0.230823-rc"/>
+      </ItemGroup>
+    </When>
+    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('ToolkitTriggersSourceProject')) == 'false'">
```

```suggestion
    <When Condition="'$(IsSingleExperimentHead)' == 'true' and $(MSBuildProjectName.StartsWith('Triggers')) == 'false'">
```
Fix the condition here.